### DISCUSSION TITLE
Cara membuat filter menggunakan terminal?

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#726] [Q&A] [answered]
Bro klo bisa request video tutorial ttg contoh cara bikin filter ads pakai terminal

Sdh coba instal di linux manjaro, sdh berhasil instal, tp blm tau gambaran bikin filternya, 

Klo manual edit filter lewat github web sdh tau caranya

**Comment by realodix** [comment]
Maksudnya gimana bro? Cara render filternya ya? Di Windows jg bisa kok

**Comment by 3xploiton3** [comment]
belum ngerti cara buat filter di terminal (cmd/power shell) kemudian di push ke github

**Comment by realodix** [comment, marked as answer]
Pada panduan yang Saya tulis di [README.md#development](https://github.com/realodix/AdBlockID/blob/master/README.md#development), yang bingung di bagian mananya Gan?

Pada dasarnya agan edit filter di `/src` lalu satukan pecahan filternya ke `/output` dengan menjalankan file `build.sh`

Untuk push ke github, agan bisa gunakan aplikasi pada https://git-scm.com/download/gui/windows

**Comment by 3xploiton3** [comment]
ok saya coba dulu, ternyata Git ada versi GUI-nya

**Comment by realodix** [comment]
Gimana Gan, ada yg masih bingung?

Ada Gan, Saya pribadi pakai GitHub Desktop

**Comment by 3xploiton3** [comment]
![image](https://user-images.githubusercontent.com/19517680/79490194-b1f65b80-8046-11ea-954e-e77eb3fc0508.png)

saya coba pakai smartgit tampilannya seperti itu, mau nulis filternya dimana?

contoh saya mau nulis filter ini

```
/*1001win*.$image
/*qqangpao*.$image
##[href*="1001win."]
##[href*="qqangpao."]
```

**Comment by realodix** [comment]
Untuk penulisannya, agan bisa gunakan editor favorit seperti pada https://www.elegantthemes.com/blog/resources/best-code-editors

Lalu kelomppokkan ke dalam masing-masing file

/src/addons/adult-block.adbl
```
/*1001win*.$image
/*qqangpao*.$image
```

/src/addons/adult-hide.adbl
```
##[href*="1001win."]
##[href*="qqangpao."]
```

**Comment by 3xploiton3** [comment]
misal saya tulis filter, apakah bisa masuk ke kelompok file secara otomatis?

jadi gak pusing harus mencari kategori file,

**Comment by realodix** [comment]
Gak bisa Gan :D

**Comment by realodix** [comment]
Sepertinya sudah tidak ada yang ditanyakan, maka akan Saya tutup.

Jika ada yang ditanyakan kembali, jangan sungkan untuk bertanya
